### ISSUE TITLE
Usability enhancements

### ISSUE THREAD

**Issue by chuckheron** [CDrummond/cantata#894]
Hi
I like Cantata a lot, but there are a extra couple buttons/features that I would love to have.

Shuffle vs Random, I prefer using shuffle playlist instead of random play, then I can see what's next in the queue... then if I don't like it, I just shuffle again.  Either a new shuffle button, or a way to swap the behavior of the Random button.

A "Play next" context menu option would be great (like `mpc insert`)

Also a way to replace the current playlist without interrupting the current song would be nice.
Basically an `mpc crop` and then `mpc insert` 

Thanks for the nice mpd client!
-chuck

**Comment by CDrummond** [comment]
You can already shuffle the play queue - its in the context memory.

You can also drag'n'drop songs onto the playqueue. Sorry, but the Add/Play menus already have enough entries and I'm not going to add more.

**Comment by chuckheron** [comment]
Ok, well thanks anyway..
 

    On Thursday, September 29, 2016 11:58 PM, CraigD <notifications@github.com> wrote:
 

 You can already shuffle the play queue - its in the context memory.You can also drag'n'drop songs onto the playqueue. Sorry, but the Add/Play menus already have enough entries and I'm not going to add more.—
